Source/OxyPlot.Wpf/PlotView.cs

```diff
@@ -125,5 +126,22 @@ private void DoCopy(object sender, ExecutedRoutedEventArgs e)
             var bitmap = exporter.ExportToBitmap(this.ActualModel);
             Clipboard.SetImage(bitmap);
         }
+
+        /// <summary>
+        /// Identifies the <see cref="TextMeasurementMethod"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty TextMeasurementMethodProperty =
+            DependencyProperty.Register(
+                nameof(TextMeasurementMethod), typeof(TextMeasurementMethod), typeof(PlotViewBase), new PropertyMetadata(TextMeasurementMethod.TextBlock));
+
+        /// <summary>
+        /// Gets or sets the vertical zoom cursor.
+        /// </summary>
+        /// <value>The zoom vertical cursor.</value>
+        public TextMeasurementMethod TextMeasurementMethod
+        {
+            get => (TextMeasurementMethod)this.GetValue(TextMeasurementMethodProperty);
+            set => this.SetValue(TextMeasurementMethodProperty, value);
+        }
```

Comment:
I think those should be at the top of the class to be consistent with our general code style.

Reply:
sure, that one is easy to fix :)
I wasn't sure where to put it as all the other properties are under OxyPlot.Wpf.Shared, but TextMeasurementMethod isn't